others/RESOURCES.md

```diff
@@ -11,3 +11,10 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Interesting documentaries about different technologies
+* [Fireship.io](https://www.youtube.com/@Fireship) - JS, TS, Code Reports, 100s
```

Comment:
`JS` -> `JavaScript`
`TS` -> `TypeScript`
`100s` -> `short educational videos`
Probably mention that he also does code report which rubric about the latest tech related news.

Reply:
Sure thing

Reply:
Done